### PR TITLE
18 research how text to speech feedback will be presented

### PR DESCRIPTION
**Pull request by ICTEnablers**

### CODE REVIEW

**Comment by arethsu** [comment]
Shouldn't this be merged into `develop`? If so, click "edit" and then select `develop` instead. :)

File: src/components/tts/tts.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import Speech from 'speak-tts';
```
Comment (duff92) [inline src/components/tts/tts.tsx]: Not really a component but rather just supporting methods used for audio. Maybe move this somewhere else in our folder structure?

File: src/components/tts/tts.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  .init({
+  volume: 0.5,
+  lang: "sv-SE",
+  rate: 1,
+  pitch: 1,
+  listeners: {
+    onvoiceschanged: voices => {
+      voices.forEach(voice => { 
+        if( voice.lang === 'sv-SE' || voice.lang === 'sv_SE') {
+          console.log('Using swedish webspeech voice: '+voice.name);
```
Comment (duff92) [inline src/components/tts/tts.tsx]: Should we keep console outputs when pushing to production? What do you say? :)

File: src/components/tts/tts.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+          speech.setVoice(voice.name);
+          isWebspeechLoaded = true;
+      }});
+    }
+  }
+})
+.then( data =>  {
```
Comment (duff92) [inline src/components/tts/tts.tsx]: Remove then statement, not doing anthing

File: src/components/tts/tts.tsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+/*TODO: Check browser support
+  const text = speech.hasBrowserSupport();_*/
+
+const webSpeech = async (text: string): Promise<string>  => {  
+    return new Promise((resolve,reject) => { 
+        speech.speak({
+        text: text,
+        queue: false,
+        listeners: {
```
Comment (duff92) [inline src/components/tts/tts.tsx]: What are these listeners doing? Redundant at this stage?

File: src/components/task/task.tsx [hunk 5]
```diff
@@ -40,36 +39,69 @@ const Task = (props) => {
     task,
     handleCorrectInput,
     handleWrongInput,
-    completed,
-    playAudio  } = props;
+    completed
+    } = props;
```
Comment (duff92) [inline src/components/task/task.tsx]: Wrong indendention, remove one tab :)

File: src/components/task/task.tsx [hunk 6]
```diff
@@ -40,36 +39,69 @@ const Task = (props) => {
...
+        text: event.key,
+      };
+
+      const nextTextToSpeak: ITTS = {
+        type: TTS_PLATTFORM.WEBSPEECH,
+        lang: 'sv-SE',
+        text: ''
+      }
+
+      if(currentPos<task.text.length-1) {
```
Comment (duff92) [inline src/components/task/task.tsx]: Add spaces in this if statement

File: src/components/tts/tts.tsx (hunk 2 above)
Comment (ICTEnablers) [reply src/components/tts/tts.tsx]: I say we keep console.error, removed .log

File: src/components/tts/tts.tsx (hunk 4 above)
Comment (ICTEnablers) [reply src/components/tts/tts.tsx]: Redundant. Removed

File: src/components/tts/tts.tsx (hunk 1 above)
Comment (ICTEnablers) [reply src/components/tts/tts.tsx]: I think we should keep it. If I did not misinterpret Joel's conversation about atomic design and components during lunch last Thursday, it should be ok to define a component this way.

**Pull request merged**